src/components/common/PeopleAndRoles/PeopleAndRoles.vue

```diff
@@ -488,7 +553,7 @@ export default class PeopleAndRoles extends Mixins(CommonMixin) {
 
     // just set the action (ie, soft-delete)
     // person will be filtered out on file and pay
-    person.action = ActionTypes.REMOVED
+    person.action = [ActionTypes.REMOVED]
```

Comment:
Property "action" should be "actions" since it's an array.

Reply:
If this messes up the schema (API side) then I recommend converting action <-> actions in the code that saves/restores the filing. In other words, put a work-around as close to the API as possible so that the rest of the UI code follows our naming scheme.

Reply:
Sorry, duplicate conversation. You can resolve one of them.

Reply:
Updated! Pluralizing a single word has never been so much work and regression testing 😆

Reply:
Yeah, you have my sympathy -- and my gratitude. It was the right thing to do :)